src/main/java/BMO/TaskList.java

```diff
@@ -0,0 +1,131 @@
+package bmo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import bmo.task.Deadline;
+import bmo.task.Event;
+import bmo.task.Task;
+import bmo.task.ToDo;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>(100);
```

Comment:
Why was this this.tasks initialised with 100?🫤 
I feel initialising arrayList with 100 is restrictive and will waste storage.
suggestion: this.tasks = new ArrayList<>();

Reply:
In the iP requirements it said to assume that the max number of tasks that the chatbot would handle is 100, thus why I initialised the ArrayList to 100.